scripts/speaker_tasks/create_alignment_manifest.py

```diff
@@ -14,39 +14,31 @@
 
 import argparse
 import os
-import random
 import shutil
+from pathlib import Path
 
 from nemo.collections.asr.parts.utils.manifest_utils import read_manifest, write_ctm, write_manifest
 from nemo.utils import logging
 
-random.seed(42)
 
-
-def get_unaligned_examples(unaligned_path, dataset):
+def get_unaligned_files(unaligned_path):
     """
-    Get librispeech examples without alignments for the desired dataset in
-    order to filter them out (as they cannot be used for data simulation))
+    Get files without alignments in order to filter them out (as they cannot be used for data simulation))
 
     Args:
         unaligned_path (str): Path to the file containing unaligned examples
-        dataset (str): LibriSpeech data split being used
 
     Returns:
         skip_files (list): Unaligned file names to skip
     """
+    skip_files = []
     with open(unaligned_path, 'r', encoding='utf-8') as f:
-        lines = f.readlines()
-        i = 0
-        skip_files = []
-        while i < len(lines):
-            l = lines[i].strip('\n')
-            if l[0] == '#':
-                unaligned_dataset = l.split(" ")[1]
-            elif unaligned_dataset == dataset:
-                unaligned_file = l.split(" ")[0]
-                skip_files.append(unaligned_file)
-            i += 1
+        for line in f.readlines():
+            line = line.strip()
+            if not line:
+                continue
+            unaligned_file = line.split()[0]
+            skip_files.append(unaligned_file)
```

Comment:
@stevehuang52 Did you fix this yourself? `unaligned_dataset == dataset` is useless so that is why it is removed right?

Reply:
Yes, I think it's not necessary and removed it.